cms/djangoapps/contentstore/views/course.py

```diff
@@ -627,6 +627,10 @@ def format_library_for_view(library):
 
     site_config = configuration_helpers.get_current_site_configuration()
     tracking_api_url = f"{site_config.get_value('PANEL_NOTIFICATIONS_BASE_URL')}/api/v1/tracking_events/"
+    frontent_redirect_url = ''
+    frontend_url = [url for url in settings.CORS_ORIGIN_WHITELIST if 'apps' in url]   
```

Comment:
@taimoor-ahmed-1 Is there better way to get panel frontend url or should we go with this for now? @Anas-hameed and I went over the code and there currently panel frontend url is not available in studio

Reply:
seems fine for now

Reply:
Alright 👍